schemas/IlpHeader.json

```diff
@@ -6,11 +6,11 @@
   "properties": {
     "ledger": {
       "description": "The ledger where the transfer will take place",
-      "$ref": "Iri.json"
+      "type": "string"
```

Comment:
Actually, that can probably be changed to use `account` and just strip off the last section.